components/utils/keyboardTrap.js

```diff
@@ -1,19 +1,33 @@
-export const nextElementKeyboardTrap = (element, firstItem, lastItem) => {
-  event.preventDefault();
+export default {
+  nextElementKeyboardTrap(element, focusableElements) {
+    event.preventDefault();
 
-  if (element !== lastItem) {
-    element.nextSibling.focus();
-  } else {
-    firstItem.focus();
-  }
-};
+    const firstItem = focusableElements[0];
```

Comment:
these two functions are pretty duplicative.  the duplication is on the harmless side though, so your call